pom.xml

```diff
@@ -554,6 +554,11 @@ limitations under the License.
                     <artifactId>maven-clean-plugin</artifactId>
                     <version>3.1.0</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
@rpopov enforcer plugin shows up twice here.  Should be merged to one.

Reply:
@robertdale, the second occurrence of maven-enforcer-plugin (above) is in the <pluginManagement> section and states plugin's version to be used in every tinkerpop module. Actually, the practice to use plugins and dependencies with different ad-hoc defined versions exists in tinkerpop and I understand the *.version properties like log4j.version defined in this pom as an attempt to unify the versions. The Maven way is to define them in <dependecyManagement> and <pluginManagement> once. Thus, I would suggest a new task - unify the version management of plugins and dependencies the Maven way.

Reply:
ah, so it is. good job.